src/cpp/session/modules/SessionConnectionsInstaller.R

```diff
@@ -406,7 +425,39 @@
    .rs.odbcBundleWriteIni(odbcinstPath, odbcinst)
 })
 
-.rs.addFunction("odbcBundleRegisterWindows", function(name, driverPath, version) {
+.rs.addFunction("odbcBundleDriverIniPath", function(name, driverPath) {
+   dir(driverPath, pattern = paste(tolower(name), ".*\\.ini", sep = ""), recursive = T, full.names = T)
```

Comment:
nit: the `.*` is redundant; `\\.ini$` would be better.

Reply:
I'm looking for a specific name since I'm pasting `name` to the regular expression, so removing `.*` would break this.